### ISSUE TITLE
Manifest not compatible with Kubernetes 1.19+

### ISSUE THREAD

**Issue by k9ert** [bitnami-labs/sealed-secrets#454]
In parallel with #269 a lot of the resources still has v1beta attached to it's apiVersion of it's [version 0.12.5](https://github.com/bitnami-labs/sealed-secrets/releases/download/v0.12.5/controller.yaml).

**Comment by ghostsquad** [comment]
It would be nice to have a flag or something to specify what version to use, if you wish to change it from default. The default could probably be whatever is compatible with current version and prior 2 versions.

**Comment by KlavsKlavsen** [comment]
and k8s 1.18 is End-Of-Life in 3 months.. So we'd like to be able to upgrade (with sealed secrets working) to 1.20 soon :)

**Comment by mkmik** [comment]
possibly this issue has not been updated with the latest fixes.

does any of the following open issues block an upgrade to 1.20?

-  ClusterRoleBinding (v1.22)
-  Role (v1.22)

**Comment by KlavsKlavsen** [comment]
so in latest v0.14.0 - CRD has been fixed to work with 1.20.. (would be very good :)

**Comment by mkmik** [comment]
sorry I don't understand what you mean here (my brain is not 100% lately). Is the latest release (v0.14.1) working for you on 1.20 or not?

**Comment by mkmik** [comment]
I added k8s v1.20.1 to the integration test matrix in https://github.com/bitnami-labs/sealed-secrets/pull/533
and I confirm it works

**Comment by mkmik** [comment]
moving the other non blocking cleanups into #534
